iclr_wrap_up/plotter/informationplane_movie.py

```diff
@@ -0,0 +1,63 @@
+import os
+import numpy as np
+
+import matplotlib.pyplot as plt
+from matplotlib.animation import FFMpegWriter
+
+from iclr_wrap_up.plotter.base import BasePlotter
+
+
+def load(run, dataset):
+    return InformationPlaneMoviePlotter(run, dataset)
+
+
+class InformationPlaneMoviePlotter(BasePlotter):
+    """Plot the infoplane movie for several runs of the same network."""
+    plotname = 'infoplane_movie'
+
+    def __init__(self, run, dataset):
+        self.dataset = dataset
+        self.run = run
+
+    def generate(self, measures_summary):
+        self.plot(measures_summary)
+        filename = f'plots/{self.plotname}.mp4'
```

Comment:
filename is duplicated here and in the call to writer. Maybe it is easiest to turn in into an attribute to avoid duplication.